CorsixTH/Src/th_gfx.cpp

```diff
@@ -28,6 +28,7 @@ SOFTWARE.
 #include <cassert>
 #include <climits>
 #include <cstring>
+#include <iostream>
```

Comment:
Leftover debugging?

Reply:
Yup, from the original feature branch :-). Removed.